spec/support/autoload/page_objects/task_list.rb

```diff
@@ -1,5 +1,8 @@
 module PageObjects
   class TaskList < SitePrism::Section
+    element :not_started_status, ".govuk-tag.govuk-tag--grey"
+    element :accepted_status, ".govuk-tag.govuk-tag--green"
```

Comment:
Rather than adding elements specific to one page in the `TaskList` page object, it's better to use the `find_item` method below. For example, something like:

```rb
item = assessor_application_page.task_list.find_item("Contact relevant competent authority")
expect(item.status_tag.text).to eq("NOT STARTED")
```

This has the benefit of implicitly checking that the right item exists in the task list at the same time.